MediaAdmin/FileAlternatives/Strategy/ImageStrategy.php

```diff
@@ -165,14 +168,15 @@ public function overrideAlternative(MediaInterface $media, $newFilePath, $format
      */
     public function cropAlternative(MediaInterface $media, $x, $y, $h, $w, $formatName)
     {
-        $alternativeName = $this->getAlternativeName($formatName, $media->getFilesystemName());
-
         $originalFilePath = $this->mediaStorageManager->downloadFile($media->getFilesystemName(), $this->tmpDir);
+        $tmpFileName = time() . '-' . $media->getName();
+        $newFilePath = $this->tmpDir . DIRECTORY_SEPARATOR . $tmpFileName;
+        rename($originalFilePath, $newFilePath);
```

Comment:
This native php function is not mockable in the unit test. Can you use instead the symfony abstraction layer Symfony\Component\Filesystem\Filesystem ?
There is a method $filesystem->rename($origin, $target);